modules/api/pages/rest-api-extension-archetype.adoc

```diff
@@ -55,96 +55,49 @@ A https://maven.apache.org/archetype/index.html[maven archetype] is a maven proj
 
 To create your Rest API Extension project, prompt a terminal and enter the following command:
 
+[source,bash]
+----
+ $ mvn archetype:generate /
```

Comment:
copy past killed you :p

There is the same issue l.107, I can't make a suggestion because it is not part of the pull request but please remove the $ aswell.

```suggestion
mvn archetype:generate /
```